### PR TITLE
SAK-42963 Messages : empty body validation

### PR DESCRIPTION
**Pull request by bgarciaentornos**
https://jira.sakaiproject.org/browse/SAK-42963

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/jsp/pvtMsgReplyAll.jsp [hunk 1]
```diff
@@ -267,9 +267,7 @@
 				</div>
 		  </div>
 
-
-
-			<h4><h:outputText value="#{msgs.pvt_message}" /></h4>
+			<h4><h:outputText value="#{msgs.pvt_star}" styleClass="reqStar"/><h:outputText value="#{msgs.pvt_message}" /></h4>
```
Comment (bgarciaentornos) [inline msgcntr/messageforums-app/src/webapp/jsp/pvtMsgReplyAll.jsp]: Aligning is off on the file, I decided to set it as the associated inputRichText

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java [hunk 2]
```diff
@@ -2731,6 +2760,14 @@ private PrivateMessage processPvtMsgReplyAllSendHelper(boolean preview, Boolean
 		  }
 		  return null ;
 	  }
+	  if(StringUtils.isEmpty(getForwardBody())) {
```
Comment (ern) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java]: does `StringUtils.isBlank()` make more sense?

Comment (bgarciaentornos) [reply msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java]: All the similar comparisons in the file are using isEmpty, maybe it's acceptable to send a blank space if you don't want to add anything else... But I can change it.

File: msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties [hunk 3]
```diff
@@ -632,6 +632,8 @@ pvt_select_msg_recipient_replyall = You must select the reply all recipient befo
 
 pvt_missing_subject = You must enter a subject before you may send this message.
 pvt_missing_subject_draft =You must enter a subject before you may save this message.
+pvt_missing_body = You must enter a body before you may send this message.
```
Comment (fostersdesign) [inline msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties]: I'd prefer to change the wording of "enter a body" to something else to make it clearer what is being asked here. Something like "You must write your message before you can send it."

**Pull request merged**